spec/01-unit/004-utils_spec.lua

```diff
@@ -479,8 +479,8 @@ describe("Utils", function()
       it("correctly formats addresses", function()
         assert.are.equal("1.2.3.4", utils.format_host("1.2.3.4"))
         assert.are.equal("1.2.3.4:80", utils.format_host("1.2.3.4", 80))
-        assert.are.equal("[::1]", utils.format_host("::1"))
-        assert.are.equal("[::1]:80", utils.format_host("::1", 80))
+        assert.are.equal("[0000:0000:0000:0000:0000:0000:0000:0001]", utils.format_host("::1"))
+        assert.are.equal("[0000:0000:0000:0000:0000:0000:0000:0001]:80", utils.format_host("::1", 80))
```

Comment:
this should be formatting only, not normalizing

Reply:
Thibault and I discussed yesterday and decided that formatting is allowed to include normalisation.